Makefile

```diff
@@ -38,6 +38,9 @@ test: license-check gofmt
 	go test -cover $(TEST_PACKAGES_EXPANDED)
 	cd dist; ./mk-docker-opts_tests.sh
 
+e2e-test: dist/flanneld-$(ARCH)-$(TAG).docker
+	cd dist; sudo -E ./functional-test.sh $(REGISTRY)/flannel-$(ARCH):$(TAG)
```

Comment:
I've made some changes to the naming of images - it's always flannel:$(TAG)-$(ARCH) now